ocient/assets/dashboards/ocient_overview.json

```diff
@@ -0,0 +1 @@
"min":"auto","max":"auto"}},"layout":{"x":0,"y":3,"width":4,"height":3}},{"id":4844547439564550,"definition":{"title":"Packets of data transmitted per interface","show_legend":false,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"alias":"Packets of data transmitted","formula":"query1"}],"response_format":"timeseries","queries":[{"query":"avg:system.net.packets_out.count{$Cluster} by {device}","data_source":"metrics","name":"query1"}],"style":{"palette":"warm","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"}},"layout":{"x":4,"y":3,"width":4,"height":3}},{"id":887623889786274,"definition":{"title":"Packet transmit errors detected by the device driver","show_legend":false,"legend_layout":"auto","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"alias":"Packet transmit errors detected","formula":"query1"}],"response_format":"timeseries","queries":[{"query":"avg:system.net.packets_out.error{$Cluster} by {device}","data_source":"metrics","name":"query1"}],"style":{"palette":"warm","line_type":"solid","line_width":"normal"},"display_type":"line"}],"yaxis":{"include_zero":true,"scale":"linear","label":"","min":"auto","max":"auto"}},"layout":{"x":8,"y":3,"width":4,"height":3}}]},"layout":{"x":0,"y":7,"width":12,"height":7}},{"id":2392989914101374,"definition":{"title":"Table Information","background_color":"vivid_blue","show_title":true,"type":"group","layout_type":"ordered","widgets":[{"id":6773250723091639,"definition":{"title":"System Fill By Table","title_size":"16","title_align":"left","show_legend":false,"legend_layout":"horizontal","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"name":"query1","data_source":"metrics","query":"avg:ocient.db.size{$Cluster} by {table_name}"}],"response_format":"timeseries","style":{"palette":"dog_classic","order_by":"values","line_type":"solid","line_width":"normal"},"display_type":"area"}]},"layout":{"x":0,"y":0,"width":6,"height":3}},{"id":1132387622919265,"definition":{"title":"Rows By Table","title_size":"16","title_align":"left","show_legend":false,"legend_layout":"horizontal","legend_columns":["avg","min","max","value","sum"],"type":"timeseries","requests":[{"formulas":[{"formula":"query1"}],"queries":[{"name":"query1","data_source":"metrics","query":"avg:ocient.db.tot_rows{$Cluster} by {schema,name}"}],"response_format":"timeseries","style":{"palette":"dog_classic","order_by":"values","line_type":"solid","line_width":"normal"},"display_type":"area"}]},"layout":{"x":6,"y":0,"width":6,"height":3}}]},"layout":{"x":0,"y":14,"width":12,"height":4}}],"template_variables":[{"name":"Cluster","prefix":"cluster","available_values":[],"default":"*"}],"layout_type":"ordered","notify_list":[],"reflow_type":"fixed","tags":[]}
```

Comment:
For the description, recommend the following:

This dashboard gives you a high-level overview of your Ocient environment performance.\nIt displays critical information to provide you a bird's-eye view of the health of your Ocient clusters.\n\nWe recommend tagging all nodes in your deployment with a `cluster` tag to easily switch widgets between clusters.\n\nFor more information on getting started with Ocient, see the [Ocient documentation](https://docs.ocient.com/)